README.md

```diff
@@ -5,9 +5,9 @@
 
 ## What is Nebulet?
 
-Nebulet is a microkernel that executes WebAssembly modules instead of ELF binaries. Furthermore, it does so in ring 0 and in the same address space as the kernel, instead of in ring 3. Normally, this would be super dangerous, but WebAssembly is designed to run safely on remote computers, so it can be securely sandboxed without losing performance.
+Nebulet is a microkernel that executes WebAssembly modules in ring 0 and a single address space to increase performance. This allows for low context-switch overhead, syscalls just being function calls, and exotic optimizations that simply would not be possible on conventional operating systems. The WebAssembly is verified and, due to a trick used to optimize out bounds-checking, unable to even represent the act of writing or reading outside its assigned linear memory.

```

Comment:
`verified and, due` should be `verified, and due`

Reply:
Thanks for catching that!